src/common/editable.js

```diff
@@ -32,5 +32,6 @@ export const update/*:type.update*/ = (model, action) =>
     clear(model) :
   action.type === "Editable.Select" ?
     select(model, action) :
-  // action.type === "Editable.Change"
-    change(model, action)
```

Comment:
I should have commented it somewhere, but else clause with commented condition was intentional. Flow checker will not type check if another action type is passed and will make sure that in else clause type is `Editable.Change`.